libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -375,9 +390,26 @@ class AP_InertialSensor
         float delta_time;
     } _hil {};
 
+    // Trim options
+    AP_Int8 _acc_body_aligned;
+    AP_Int8 _trim_option;
+
     DataFlash_Class *_dataflash;
 
     static AP_InertialSensor *_s_instance;
+    AP_AccelCal* _acal;
```

Comment:
why are these not allocated when they are needed instead of keeping them around when we are not doing a calibration?